From 3e1a956efc2e2dbdf0ee8ced878982287d6815b2 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Fri, 17 Nov 2017 10:52:00 +0800
Subject: [PATCH 3069/5242] MLK-16715-3 ARM64: dts: fsl-imx8qm: change USB2
 controller properties

commit  08b63c6b038c6eb0ecfce8372c1ff4a73b8bf19c from
https://source.codeaurora.org/external/imx/linux-imx.git

The imx8qm's non-core register controller is more like imx7ulp than
imx7d, since imx7d uses Samsung PHY, but imx7ulp and imx8qm use
freescale PHY, so imx8qm uses the same compatible for imx7ulp.

But imx8qm and imx7ulp's platform are so many differences, so the
compatible for driver are different.

Besides, we add performance tuning parameters and delete properties which
is dedicated for imx7d.

Acked-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |   10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 82a5124..0fa93cc 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -3000,7 +3000,7 @@
 
 	usbmisc1: usbmisc@5b0d0200 {
 		#index-cells = <1>;
-		compatible = "fsl,imx7d-usbmisc", "fsl,imx6q-usbmisc";
+		compatible = "fsl,imx7ulp-usbmisc", "fsl,imx6q-usbmisc";
 		reg = <0x0 0x5b0d0200 0x0 0x200>;
 	};
 
@@ -3020,17 +3020,19 @@
 	};
 
 	usbotg1: usb@5b0d0000 {
-		compatible = "fsl,imx7d-usb", "fsl,imx27-usb";
+		compatible = "fsl,imx8qm-usb", "fsl,imx27-usb";
 		reg = <0x0 0x5b0d0000 0x0 0x200>;
 		interrupt-parent = <&wu>;
 		interrupts = <GIC_SPI 267 IRQ_TYPE_LEVEL_HIGH>;
 		fsl,usbphy = <&usbphy1>;
 		fsl,usbmisc = <&usbmisc1 0>;
 		clocks = <&clk IMX8QM_USB2_OH_AHB_CLK>;
-		phy-clkgate-delay-us = <400>;
-		status = "disabled";
+		ahb-burst-config = <0x0>;
+		tx-burst-size-dword = <0x10>;
+		rx-burst-size-dword = <0x10>;
 		#stream-id-cells = <1>;
 		power-domains = <&pd_conn_usbotg0>;
+		status = "disabled";
 	};
 
 	usbotg3: cdns3@5b110000 {
-- 
1.7.9.5

